app/assets/javascripts/angular/controllers/orders_controller.js

```diff
@@ -41,11 +41,17 @@ app.controller('DriversController', ['$scope', '$http', function($scope, $http)
   });
 
   dispatcher.bind('get_new_order', function(data) {
-    $http.get('/drivers/orders.json').success(function(data){
-      $scope.orders = data;
-    });
+    $scope.get_orders(1000);
   });
 
+  $scope.get_orders = function(delay) {
+    setTimeout(function() {
+      $http.get('/drivers/orders.json').success(function(data){
+        $scope.orders = data;
+      });
+    }, delay);
+  }
```

Comment:
**First of all, Front-End, as I know, is my area of responsibility.** 
The second thing, as I remember we agreed that websockets will send to ui side object which was changed instead of message like we do now, therefore we don't need to make this request at all. 
And eventually, as I understand we should trigger event when the data was successfully saved to DB, because your changes doesn't have any sense in case if exception was accured during saving data to DB. 
More than that using setTimeout is bad practice when you want to do something in future but you don't exactly know when you will be able to do that, and I think this is exactly our case.